edb/lib/sys.edgeql

```diff
@@ -155,3 +155,12 @@ sys::get_version_as_str() -> std::str
         ) IF v.stage != <sys::version_stage>'final' ELSE ''
     $$;
 };
+
+
+CREATE FUNCTION
+sys::get_transaction_isolation() -> std::str
```

Comment:
do you want to make it an enum?

Reply:
Not really.  It makes no difference output-wise, and it's not an argument